### DISCUSSION TITLE
Fail gracefully if only() filter is used in CI/CD

### DISCUSSION THREAD

**Discussion by jansgescheit** [pestphp/pest#1128] [Ideas]
I just realized that our test pipeline went through so quickly.

The trigger was that the filter `->only()` made it into the our develop branch. 

Is there a way to intercept this? Ideally, Pest should fail in the CI/CD when a test is called with `->only()`.

**Comment by nunomaduro** [comment]
Doesn't `--ci` avoid this?

**Comment by jansgescheit** [comment]
yeah `--ci` avoid this. Thanks i must be blind, I didn't find it in the documentation
